### PR TITLE
Added test-lambda to installer

### PR DESCRIPTION
**Pull request by hks93**
For adding test-lambda API to jazz installer

### CODE REVIEW

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 
 #-------------------------------------------
 
-platform_services=("jazz_cognito-authorizer" "jazz_logs" "jazz_usermanagement" "jazz_services-handler" "jazz_events" "jazz_services" "jazz_logout" "jazz_login" "jazz_cloud-logs-streamer" "jazz_is-service-available" "jazz_delete-serverless-service" "jazz_create-serverless-service" "jazz_email" "jazz_events-handler" "jazz_environments" "jazz_scm-webhook" "jazz_environment-event-handler" "jazz_deployments" "jazz_deployments-event-handler")
+platform_services=("jazz_cognito-authorizer" "jazz_logs" "jazz_usermanagement" "jazz_services-handler" "jazz_events" "jazz_services" "jazz_logout" "jazz_login" "jazz_cloud-logs-streamer" "jazz_is-service-available" "jazz_delete-serverless-service" "jazz_create-serverless-service" "jazz_email" "jazz_events-handler" "jazz_environments" "jazz_scm-webhook" "jazz_environment-event-handler" "jazz_deployments" "jazz_deployments-event-handler" "test-lambda")
```
Comment (supritat) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Include as "jazz_test-lambda"

File: installscripts/jazz-terraform-unix-noinstances/scripts/scmpush.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ jazz_ui_trigger_job_url="/project/jazz_ui"
 gitlab_webhook_url="http://$jenkins_user:$jenkins_password@$jenkins_elb$gitlab_trigger_job_url"
 gitlab_jazz_ui_webhook_url="http://$jenkins_user:$jenkins_password@$jenkins_elb$jazz_ui_trigger_job_url"
 
-platform_services=("jazz_cognito-authorizer" "jazz_logs" "jazz_usermanagement" "jazz_services-handler" "jazz_events" "jazz_services" "jazz_logout" "jazz_login" "jazz_cloud-logs-streamer" "jazz_is-service-available" "jazz_delete-serverless-service" "jazz_create-serverless-service" "jazz_email" "jazz_events-handler" "jazz_environments" "jazz_scm-webhook" "jazz_environment-event-handler" "jazz_deployments" "jazz_deployments-event-handler")
+platform_services=("jazz_cognito-authorizer" "jazz_logs" "jazz_usermanagement" "jazz_services-handler" "jazz_events" "jazz_services" "jazz_logout" "jazz_login" "jazz_cloud-logs-streamer" "jazz_is-service-available" "jazz_delete-serverless-service" "jazz_create-serverless-service" "jazz_email" "jazz_events-handler" "jazz_environments" "jazz_scm-webhook" "jazz_environment-event-handler" "jazz_deployments" "jazz_deployments-event-handler" "test-lambda")
```
Comment (supritat) [inline installscripts/jazz-terraform-unix-noinstances/scripts/scmpush.sh]: Include as "jazz_test-lambda"

**Comment by suryajak** [comment]
Resolve conflicts. Also given we are about to close down v1.5.2 for release, this should go in the next release (v1.7.1.)

**Comment by hks93** [comment]
@suryajak Have resolved the merge conflicts

**Pull request merged**